docs/source/export.rst

```diff
@@ -291,6 +291,176 @@ torch.export>`), but seeing as the context manager does not affect the tensor
 computations in the model, we can go with the non-strict mode's result.
 
 
+.. _Training Export:
+
+Training Export
+^^^^^^^^^^^^^^^^^
+
+In PyTorch 2.5, we introduced a new API called :func:`export_for_training`.
+It's still going through hardening, so if you run into any issues, please file
+them to Github with the "oncall: export" tag.
+
+In this API, we produce the most generic IR that contains all ATen operators
+(including both functional and non-functional) which can be used to train in
+eager PyTorch Autograd. This API is intended for PT2 quantization training use cases
+and will soon be the default IR of torch.export.export in the near future. To read further about
+the motivation behind this change, please refer to
+https://dev-discuss.pytorch.org/t/why-pytorch-does-not-need-a-new-standardized-operator-set/2206
+
+With this API, and :func:`run_decompositions()`, you should be able to get any inference IR with
```

Comment:
Edited:

When this API is combined with :func:`run_decompositions()`, you should be able to get inference IR with
any desired decomposition behavior.